Formula/rdkit.rb

```diff
@@ -0,0 +1,90 @@
+class Rdkit < Formula
+  desc "Interpreted, interactive, object-oriented programming language"
+  homepage "http://rdkit.org/"
+  url "https://github.com/rdkit/rdkit/archive/Release_2020_09_4.tar.gz"
+  sha256 "9e734ca8f99d8be1ef2ac51efb67c393c62e88b98cfa550d6173ce3eaa87b559"
+  license "BSD-3-Clause"
+
+  head do
+    url "https://github.com/rdkit/rdkit.git"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "boost"
+  depends_on "boost-python3"
+  depends_on "eigen"
+  depends_on "freetype"
+  depends_on "numpy"
+  depends_on "postgresql@12"
+  depends_on "py3cairo"
+  depends_on "python@3.9"
+
+  def install
+    ENV["CFLAGS"] = "-Wno-parentheses -Wno-logical-op-parentheses -Wno-format"
+    ENV["CXXFLAGS"] = "-std=c++11 -Wno-parentheses -Wno-logical-op-parentheses -Wno-format"
+    cpp_version_str = `c++ --version`
+    if cpp_version_str.include? "clang"
+      puts "detected clang C++ compiler"
+      ENV["CXXFLAGS"] += " -stdlib=libc++"
+    else
+      puts "assuming GCC C++ compiler"
+      # g++ doesn"t accept/need the option: -stdlib=libc++
+    end
+
+    # Get Python location
+    python3 = Formula["python@3.9"].opt_bin/"python3"
+    python_include =
+      Utils.safe_popen_read(python3, "-c", "from distutils import sysconfig;print(sysconfig.get_python_inc(True))")
+           .chomp
+    python_executable = Utils.safe_popen_read(python3, "-c", "import sys;print(sys.executable)").chomp
+    numpy_include = Utils.safe_popen_read(python3, "-c", "import numpy ; print(numpy.get_include())").chomp
+
+    args = std_cmake_args + %W[
+      -DRDK_INSTALL_INTREE=OFF
+      -DRDK_BUILD_SWIG_WRAPPERS=OFF
+      -DRDK_BUILD_AVALON_SUPPORT=ON
+      -DRDK_BUILD_PGSQL=ON
+      -DRDK_PGSQL_STATIC=ON
+      -DMAEPARSER_FORCE_BUILD=ON
+      -DCOORDGEN_FORCE_BUILD=ON
+      -DRDK_BUILD_INCHI_SUPPORT=ON
+      -DRDK_BUILD_CPP_TESTS=OFF
+      -DRDK_INSTALL_STATIC_LIBS=OFF
+      -DRDK_BUILD_CAIRO_SUPPORT=ON
+      -DRDK_BUILD_YAEHMOP_SUPPORT=ON
+      -DRDK_BUILD_FREESASA_SUPPORT=ON
+      -DBoost_NO_BOOST_CMAKE=ON
+      -DPYTHON_INCLUDE_DIR=#{python_include}
+      -DPYTHON_EXECUTABLE=#{python_executable}
+      -DPYTHON_NUMPY_INCLUDE_PATH=#{numpy_include}
+    ]
+
+    system "cmake", ".", *args
+    system "make", "-j#{ENV.make_jobs}"
```

Comment:
```suggestion
    system "make"
```
`brew` will specify the correct `-j` flag on its own.